src/generators/web.cr

```diff
@@ -45,10 +45,9 @@ class LuckyCli::Generators::Web
       add_browser_authentication_to_src
     end
 
-    # TODO: Add API auth with JWT
-    # if api_only? && generate_auth?
-    #   add_api_authentication_to_src
-    # end
+    if api_only? && generate_auth?
+      add_api_authentication_to_src
+    end
 
```

Comment:
So I think my original todo snippet was wrong. This could be generated even without api_only? Because you’d want the API auth  in browser apps too.

We can make it optional in the “init” task later but I think that makes the most sense. What do you think?

Reply:
Okay that means that there would be normal (session) authentication and jwt authentication in the browser app right? I guess it does make sense since the api will need authentication too

Reply:
I see what you mean now. If we have the api being generated for the browser apps then it should get authentication generated too. I made the change and pushed up  to see if the tests pass